src/interface.jl

```diff
@@ -627,6 +824,41 @@ function Base.show(io::IO, G::LieGroup)
     return print(io, "LieGroup($(G.manifold), $(G.op))")
 end
 
+_doc_vee = """
+    vee(G::LieGroup, X)
+    vee!(G::LieGroup, c, X)
+
+Compute the vee map ``(⋅)^∨`` that maps a tangent vector `X` from the [`LieAlgebra`](@ref)
+to its coordinates with respect to a certain bvasis in the Lie algebra
+
+```math
+X = $(_tex(:sum))_{i∈$(_tex(:Cal,"I"))} c_iB_i,
```

Comment:
Maybe we could use {1, 2, ..., n} for simplicity instead of an index set? This function is only for finite-dimensional Lie groups after all.

Reply:
currently it is, so I wanted to be open for that, but sure we can also change that.

Reply:
For now I mentioned what the index set usually is. Will leave this open to be resolved by you if this is ok. That way we could easier move to the infinite dimensional case, which I might indeed explore at some point.

Reply:
OK. By the way, in the inifinite dimensional case all we can really do is picking some finite-dimensional subspace to represent, though that subspace is often adapted to the data we want to represent.

Reply:
Numerically probably, mathematically not :)